### PR TITLE
Empty Compose M3 template

### PR DESCRIPTION
**Pull request by android-dev-lxl**
M3 template code
Copyright
Updated theme

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -49,16 +64,16 @@ android {
...
     implementation "androidx.compose.ui:ui-tooling"
     implementation "androidx.compose.ui:ui-tooling-preview"
     implementation 'androidx.core:core-ktx:1.9.0'
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
-    implementation "androidx.lifecycle:lifecycle-viewmodel-compose:2.5.1"
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.6.1'
+    implementation "androidx.lifecycle:lifecycle-viewmodel-compose:2.6.1"
     implementation "androidx.navigation:navigation-compose:2.5.3"
 
     //Room
```
Comment (johnshea) [inline app/build.gradle:79]: Should the Room dependencies be under their respective sections?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -29,6 +28,7 @@
         <activity
             android:name=".MainActivity"
             android:exported="true"
+            android:label="@string/app_name"
             android:theme="@style/Theme.Inventory">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
```
Comment (johnshea) [inline app/src/main/AndroidManifest.xml:34]: Remove the following blank line.

**Review by johnshea — APPROVED** [review]
lgtm. small nit/question

File: app/build.gradle (hunk 1 above)
Comment (android-dev-lxl) [reply app/build.gradle:79]: This is optional. Either way is fine. 
As long as we are consistent in the app.

**Pull request merged**